docs/_articles/ja/apex/apex-debugging.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Debugging Apex
+lang: en
+---
+
+The Salesforce VS Code Extensions pack comes with rich debugging options for all levels of developer experiences. Use this information to figure out which Apex debugger is right for you:
+
+- [Replay Debugger](replay-debugger.md): Free and easy to use.
```

Comment:
Should we include some more info on the Replay Debugger here?

Reply:
Sure. Let me know what else I could add?

Reply:
Realized we have the other sections with more details :)